packages/client/src/components/Order-Page/OrderPage.component.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import ShoppingCart from '../ShoppingCart/ShoppingCart.component';
+import DeliveryDetails from '../DeliveryDetails/DeliveryDetails.component';
+import Carousel from '../Carousel/Carousel.component';
+import getApiBaseUrl from '../../utils/getApiBaseUrl';
+import './OrderPage.css';
+
+function OrderPage() {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    fetchUser();
+  }, []);
+
+  const fetchUser = () => {
+    fetch(`${getApiBaseUrl()}/api/orders`, {
+      method: 'GET',
+      headers: { 'Content-Type': 'application/json' },
+    })
+      .then((res) => res.json())
+      .then((result) => {
+        setData(result);
+      });
+  };
+
+  return (
+    <div>
+      <div className="order-page">
+        <p className="order-page-path">
+          Simply Spices / Shopping Cart and Delivery Details{' '}
```

Comment:
```suggestion
          Simply Spices / Shopping Cart and Delivery Details
```

Use CSS margin instead